src/test/java/marquez/api/mappers/NamespaceMapperTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package marquez.api.mappers;
+
+import static marquez.api.models.ApiModelGenerator.newNamespaceRequest;
+import static marquez.common.models.CommonModelGenerator.newNamespaceName;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatNullPointerException;
+
+import java.util.Optional;
+import marquez.UnitTests;
+import marquez.api.models.NamespaceRequest;
+import marquez.common.models.NamespaceName;
+import marquez.service.models.Namespace;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+@Category(UnitTests.class)
+public class NamespaceMapperTest {
+  private final NamespaceName NAMESPACE_NAME = newNamespaceName();
+  private final NamespaceRequest REQUEST = newNamespaceRequest();
```

Comment:
For performance reasons, these can be static. JUnit creates a new instance of the class for every test method, so each time, we'd need to instantiate these. By making it static, we avoid those extra instantiations.
```
Every time we run a JUnit test class, a new instance of it is created. There is even a new instance before each test method execution. JUnit framework provides following basic lifecycle annotations (all are method level annotations).
```
source: https://www.logicbig.com/tutorials/unit-testing/junit/lifecycle.html

Reply:
They should be defined as constants, just forgot to make them static. Nice catch!